File: sdks/java/lib/src/main/java/models/shared/FileVersionLegalHolds.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * FileVersionLegalHolds
 * The part of an API response that describes marker
 * based pagination
**/
public class FileVersionLegalHolds {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("entries")
    public FileVersionLegalHold[] entries;
    public FileVersionLegalHolds withEntries(FileVersionLegalHold[] entries) {
        this.entries = entries;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("limit")
    public Long limit;
    public FileVersionLegalHolds withLimit(Long limit) {
        this.limit = limit;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("next_marker")
    public Long nextMarker;
    public FileVersionLegalHolds withNextMarker(Long nextMarker) {
        this.nextMarker = nextMarker;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("prev_marker")
    public Long prevMarker;
    public FileVersionLegalHolds withPrevMarker(Long prevMarker) {
        this.prevMarker = prevMarker;
        return this;
    }
}

File: python-sdk/src/boxsdk/models/shared/fileversionlegalholds.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import fileversionlegalhold as shared_fileversionlegalhold


@dataclass_json
@dataclasses.dataclass
class FileVersionLegalHolds:
    r"""FileVersionLegalHolds
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_fileversionlegalhold.FileVersionLegalHold]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    
